### PR TITLE
feat: Create new Institution without party registry [SELC-1661]

### PR DESCRIPTION
**Pull request by andrea-putzu**

### CODE REVIEW

**Review by galales — APPROVED** [review]

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -714,6 +714,45 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/institutions/insert/{externalId}':
+    post:
+      tags:
+        - process
+      summary: Create an institution using external institution id without fetching data from user-registry
```
Comment (beetlecrunch) [inline src/main/resources/interface-specification.yml]:
```suggestion
      summary: Create an institution using external institution id without fetching data from party-registry
```

File: src/main/resources/interface-specification.yml [hunk 2]
```diff
@@ -714,6 +714,45 @@ paths:
...
+      description: Create an institution using external institution id without fetching data from user-registry
```
Comment (beetlecrunch) [inline src/main/resources/interface-specification.yml]:
```suggestion
      description: Create an institution using external institution id without fetching data from party-registry
```

File: src/main/resources/interface-specification.yml [hunk 3]
```diff
@@ -714,6 +714,45 @@ paths:
...
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InstitutionSeed'
+      responses:
+        '201':
```
Comment (beetlecrunch) [inline src/main/resources/interface-specification.yml]: If you want to retrieve the object created 200 is the right choice.
201 is used together with the Location header.

File: src/test/scala/it/pagopa/interop/partyprocess/PartyApiSpec.scala [hunk 4]
```diff
@@ -1238,6 +1237,177 @@ trait PartyApiSpec
...
+          )
+        ),
+        dataProtectionOfficer = Option(
+          DataProtectionOfficer(
+            address = Option("address1"),
+            email = Option("email1@where.com"),
+            pec = Option("pec1@where.com")
+          )
+        )
+      )*/
```
Comment (beetlecrunch) [inline src/test/scala/it/pagopa/interop/partyprocess/PartyApiSpec.scala]: Is this a helpful comment or can it be deleted?

Comment (andrea-putzu) [reply src/test/scala/it/pagopa/interop/partyprocess/PartyApiSpec.scala]: no, isn't helpful

**Pull request merged**